src/scripts/main.js

```diff
@@ -1 +1,91 @@
 'use strict';
+
+const notification = document.createElement('div');
+
+notification.setAttribute('data-qa', 'notification');
+
+const promise1 = new Promise((resolve, reject) => {
+  let clickHappened = false;
+
+  const clickHandle = () => {
+    const textResolve = 'First promise was resolved';
+
+    notification.append(textResolve);
+
+    resolve(notification);
+    clickHappened = true;
+
+    document.removeEventListener('click', clickHandle);
+  };
+
+  document.addEventListener('click', clickHandle);
+
+  setTimeout(() => {
+    if (!clickHappened) {
+      const textReject = 'First promise was rejected';
+
+      notification.append(textReject);
+      reject(notification);
+    }
+  }, 3000);
+});
+
+const promise2 = new Promise((resolve) => {
+  const clickHandle = () => {
+    const textResolve = 'Second promise was resolved';
+
+    notification.append(textResolve);
```

Comment:
Appending text directly to the notification element may lead to concatenated messages if multiple promises resolve. Consider clearing or updating the content instead of appending.